readme.md

```diff
@@ -171,30 +177,31 @@ Once installation is working, try creating and requesting resources. Note that t
 * **Question:** You may notice that you can add custom error messages to individual fields failing. Try adding these and take a look at the error message received. How can you make use of those specific messages for each field?
 
 * **Your Answer:**
-
+By adding a constraint failure message to the schema definition: required:[true, 'a start year is required']
 ---
 
-- [ ] With Mongo, it is simple to create complex data structures. Add a `characters` field to your Series model that is an array of objects. Each object should have a `name` and `image_url`. Only the `name` field on the character should be required. _Note: Don't forget to change your select statements to include the `characters` field!_
+- [x] With Mongo, it is simple to create complex data structures. Add a `characters` field to your Series model that is an array of objects. Each object should have a `name` and `image_url`. Only the `name` field on the character should be required. _Note: Don't forget to change your select statements to include the `characters` field!_
 
 * **Question:** Take a look at the response from making a new series with a character. What field was created that you did not have to define?
 
 * **Your Answer:**
+_id
 
-* **Question:** With the current routes that we have, how would you upate the name of a character in a series?
+* **Question:** With the current routes that we have, how would you update the name of a character in a series?
 
 * **Your Answer:**
-
+By adding characters:req.body.characters to the second object parameter of findOneAndUpdate in the Put request.
 ---
 
-- [ ] While we can now update [subdocuments](https://mongoosejs.com/docs/subdocs.html), it is difficult to make changes that only relate to a single subdocument. To do so, we should make a new set of routes that relates to characters. Start by creating a `GET ALL` route for characters. The route will look something like the following and will return only the list of characters:
+- [x] While we can now update [subdocuments](https://mongoosejs.com/docs/subdocs.html), it is difficult to make changes that only relate to a single subdocument. To do so, we should make a new set of routes that relates to characters. Start by creating a `GET ALL` route for characters. The route will look something like the following and will return only the list of characters:
   ```
   GET /api/series/:seriesID/characters
   ```
 
 * **Question:** Where did you decide to put this route and why?
 
 * **Your Answer:**
-
+With the other get requests, after the :id one, to keep things organized. The order doesn't seem to make a difference.
```

Comment:
This is a fine answer, but please note that the order _can_ make a difference. For example:
```
app.get('/series/:id', ...)
app.get('/series/random', ...)
```
If you had both of these routes, you would never be able to reach `/series/random`. Instead, it would always get caught in `/series/:id` where `id` would be the string `random`.